include/qpid/dispatch/server.h

```diff
@@ -305,6 +305,12 @@ typedef struct qd_server_config_t {
      */
     char *ciphers;
 
+    /**
+     * This list is a space separated string of the allowed TLS protocols. The current possibilities are TLSv1 TLSv1.1 TLSv1.2.
```

Comment:
A possible enhancement is to sub spaces for commas and document it as a comma-separated list.